monkestation/code/modules/veth_misc_items/canopics/canopics.dm

```diff
@@ -0,0 +1,130 @@
+	icon_state = "canopic_jackal"
+	inhand_icon_state = "canopic_jackal"
+	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
+	resistance_flags = FIRE_PROOF
+	drop_sound = 'monkestation/code/modules/veth_misc_items/canopics/sounds/canopic_drop.ogg'
+	pickup_sound = 'monkestation/code/modules/veth_misc_items/canopics/sounds/canopic_pickup.ogg'
+	foldable_result = FALSE
+/datum/crafting_recipe/canopic_jackal
+	name = "jackal canopic jar"
+	result = /obj/item/storage/box/canopic_jackal
+	time = 2 SECONDS
+	tool_paths = FALSE
+	reqs = list(
+		/obj/item/stack/sheet/sandblock = 1,
+		/obj/item/stack/sheet/mineral/wood = 1,
+		/obj/item/stack/sheet/leather = 1,
+		/obj/item/stack/sheet/mineral/gold = 1,
+		/obj/item/stack/sheet/mineral/silver = 1)
+	category = CAT_CONTAINERS
+//human canopic sprited by twiggy, coded by veth
+/obj/item/storage/box/canopic_human
+	name = "human canopic jar"
+	desc = "An ornate stone canopic, inscribed with ancient hieroglyphs. These used to be used to store organs."
+	icon = 'monkestation/code/modules/veth_misc_items/canopics/icons/canopic.dmi'
+	icon_state = "canopic_human"
+	inhand_icon_state = "canopic_human"
+	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
+	resistance_flags = FIRE_PROOF
+	drop_sound = 'monkestation/code/modules/veth_misc_items/canopics/sounds/canopic_drop.ogg'
+	pickup_sound = 'monkestation/code/modules/veth_misc_items/canopics/sounds/canopic_pickup.ogg'
+	foldable_result = FALSE
+/datum/crafting_recipe/canopic_human
+	name = "human canopic jar"
+	result = /obj/item/storage/box/canopic_human
+	time = 2 SECONDS
+	tool_paths = FALSE
+	reqs = list(
+		/obj/item/stack/sheet/sandblock = 1,
+		/obj/item/stack/sheet/mineral/wood = 1,
+		/obj/item/stack/sheet/leather = 1,
+		/obj/item/stack/sheet/mineral/gold = 1,
+		/obj/item/stack/sheet/mineral/silver = 1)
+	category = CAT_CONTAINERS
+//monke canopic sprited by twiggy, coded by veth
+/obj/item/storage/box/canopic_monke //creates the object
+	name = "monke canopic jar"
+	desc = "An ornate stone canopic, inscribed with ancient hieroglyphs. These used to be used to store organs."
+	icon = 'monkestation/code/modules/veth_misc_items/canopics/icons/canopic.dmi'
+	icon_state = "canopic_monke"
+	inhand_icon_state = "canopic_monke"
+	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
+	resistance_flags = FIRE_PROOF
+	drop_sound = 'monkestation/code/modules/veth_misc_items/canopics/sounds/canopic_drop.ogg'
+	pickup_sound = 'monkestation/code/modules/veth_misc_items/canopics/sounds/canopic_pickup.ogg'
+	foldable_result = FALSE
+/obj/item/storage/box/canopic_monke/Initialize(mapload)
+	. = ..()
+	atom_storage.max_specific_storage = WEIGHT_CLASS_TINY
```

Comment:
Check if these fit in pockets or not tiny might mean they do

Reply:
they do, gonna only take organs when I'm done with it with 1 slot